dashboards-observability/public/components/explorer/home.tsx

```diff
@@ -114,6 +117,55 @@ export const Home = (props: IHomeProps) => {
   const handleQueryChange = async (query: string, index: string) => setSearchQuery(query);
 
   const handleTimePickerChange = async (timeRange: Array<string>) => setSelectedDateRange(timeRange);
+
+  const addSavedQueryInput = async (
+    tabId: string,
+    searchQuery: string,
+    selectedDateRange: [],
+    selectedTimeStamp: string
+  ) => {
+    console.log("tabID: ", tabId);
+    console.log("search query: ", searchQuery);
+    console.log("time range: ", selectedDateRange);
```

Comment:
We should remove console.log statements.

Reply:
I'll remove them